### PR TITLE
feat: Add support for Quasar Framework

### PR DESCRIPTION
**Pull request by rstoenescu**

### CODE REVIEW

File: server/src/modes/template/tagProviders/index.ts [hunk 1]
```diff
@@ -68,6 +71,22 @@ export function getTagProviderSettings(workspacePath: string | null | undefined)
     if (packageJson.dependencies['vuetify']) {
       settings['vuetify'] = true;
     }
+
+    const quasarPath = ts.findConfigFile(
```
[the remark this answers is not in the document]
Comment (rstoenescu) [reply server/src/modes/template/tagProviders/index.ts]: This is best for all the ways Quasar can be consumed (through Quasar CLI, vue-cli-plugin-quasar, etc etc). Explained on private.

**Comment by octref** [comment]
Need to fix Travis which uses node 6, as Quasar depends on Node 8.

This is fine for Vetur since in next release of VS Code we jump to Node 8.9.3 with Electron 2.

**Comment by TonyPythoneer** [comment]
What is it beneficial for this?

**Comment by octref** [comment]
@TonyPythoneer When you install `quasar` dependency, you get IntelliSense for its components:

image

**Pull request merged**